src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode
```

Comment:
Where are you using these methods?

Reply:
no 😞

Reply:
I'm not using equals and hashcode here but getters and setters i use in DTO mappers to set fields